website/docs/Pipelines/BuildIndices_Pipeline/README.md

```diff
@@ -0,0 +1,122 @@
+| genome_fa | Genome FASTA file used for building indices. | File |
+| biotypes | TSV file containing gene biotypes attributes to include in the modified GTF file; the first column contains the biotype and the second column contains “Y” to include or “N” to exclude the biotype; [GENCODE biotypes](https://www.gencodegenes.org/pages/biotypes.html) are used for GENCODE references and RefSeq biotypes are used for NCBI references. | File |
+
+## BuildIndices tasks and tools
+
+Overall, the BuildIndices workflow:
+1. Checks inputs, modifies reference files, and creates STAR index.
+2. Calculates chromosome sizes.
+3. Builds reference bundle for bwa.
+
+The tasks and tools used in the BuildIndices workflow are detailed in the table below. 
+
+To see specific tool parameters, select the [workflow WDL link](https://github.com/broadinstitute/warp/blob/master/pipelines/skylab/build_indices/BuildIndices.wdl); then find the task and view the `command {}` section of the task in the WDL script. To view or use the exact tool software, see the task's Docker image which is specified in the task WDL `# runtime values` section as `docker: `.
+
+| Task name | Tool | Software | Description | 
+| --- | --- | --- | --- | 
+| BuildStarSingleNucleus | [modify_gtf.py](https://github.com/broadinstitute/warp-tools/blob/develop/3rd-party-tools/build-indices/modify_gtf.py), STAR | [warp-tools](https://github.com/broadinstitute/warp-tools/tree/develop), [STAR](https://github.com/alexdobin/STAR) | Checks that the input GTF file contains input genome source, genome build version, and annotation version with correct build source information, modifies files for the STAR aligner, and creates STAR index file. |
+| CalculateChromosomeSizes | faidx | [Samtools](http://www.htslib.org/) | Reads the genome FASTA file to create a FASTA index file that contains the genome chromosome sizes. |
+| BuildBWAreference | index | [bwa-mem2](https://github.com/bwa-mem2/bwa-mem2) | Builds the reference bundle for the bwa aligner. |
+
+#### 1. Check inputs, modify reference files, and create STAR index file
+
+**Check inputs**
+
+The BuildStarSingleNucleus task reads the input GTF file and verifies that the `genome_source`, `genome_build`, and `gtf_annotation_version` listed in the file match the input values provided to the pipeline.
+
+**Modify reference files and create STAR index**
+
+The BuildStarSingleNucleus task uses a custom python script, `[modify_gtf.py](https://github.com/broadinstitute/warp-tools/blob/develop/3rd-party-tools/build-indices/modify_gtf.py)`, and a list of biotypes ([example](https://github.com/broadinstitute/warp-tools/blob/develop/3rd-party-tools/build-indices/Biotypes.tsv)) to filter the input GTF file for only the biotypes indicated in the list with the value “Y” in the second column. The defaults in the custom code produce reference outputs that are similar to those built with 10x Genomics reference scripts.
```

Comment:
This link (modify_gtf.py](https://github.com/broadinstitute/warp-tools/blob/develop/3rd-party-tools/build-indices/modify_gtf.py) isn't rendering when I run the website. I'm not sure why - it looks correct. Not a big deal if we push it up, but I just wanted to flag it.

Reply:
Thanks for catching that! I probably put the backticks in the wrong spot. I'll take a look.